addons/medical_treatment/functions/fnc_handleBandageOpening.sqf

```diff
@@ -79,26 +80,30 @@ if (!_exist) then {
 
 _target setVariable [QEGVAR(medical,bandagedWounds), _bandagedWounds, true];
 
+// _reopeningChance = 1;
```

Comment:
Remove comments

Reply:
this just makes it easier to debug wound opening
I'd prefer to keep for now, maybe remove when we get closer to release